kb/deadline-exceeded-troubleshooting.md

```diff
@@ -0,0 +1,101 @@
+---
+slug: deadline-exceeded-troubleshooting
+title: DeadlineExceeded error troubleshooting
+tags:
+  - kb-article
+date: 2022-11-01T00:00:00Z
+---
+
+`DeadlineExceeded` is a Context error thrown when requests are not completed on time.
+
+Many requests are sent and received in a Temporal system.
+These requests can be made between services, between client and server, and between the entire system and a network.
+This extensive coverage, along with the error's brief message, makes it confusing to figure out where the error could come from.
+
+Based on what users have seen, we can deduce that the three most common problem areas lie in downed services, faulty Workflow logic, and unusually high latencies.
+
+Therefore, the best route for troubleshooting lies in checking on the 3 L's:
+
+- Logs
+- Logic
+- Latency
+
+If you're still unable to find the cause of your timeouts, please visit the community forum, community Slack, or file a support ticket if you are using Temporal Cloud.
+
+:::note
+
+Cloud users cannot access some of the logs needed to diagnose the source of the error.
+If you're using Temporal Cloud, you must file a service ticket when this error occurs.
+
+:::
+
+<!-- TODO: move the note above or delete if not needed -->
+
+### Logs
```

Comment:
These could just be bold.
They don't have to be headers.

Reply:
I could keep moving things around. 
ex) instead of listing, then explaining, kind of blend it together

Reply:
^^
With how much there is to tell, it might be better to keep the sections. I'll take a look within each support thread and figure out how to link to a specific thing—especially if we first link to the user problem and then later to the steps taken